iperf.py

```diff
@@ -26,14 +26,14 @@
 
 class IperfServer(perf.PerfServer):
     def __init__(self, tc: TestConfig, ts: TestSettings):
+        super().__init__(tc, ts)
+
+        self.exec_persistent = ts.conf_server.persistent
 
-        self.exec_persistent = ts.server_is_persistent
         if self.exec_persistent:
             self.template_args["command"] = IPERF_EXE
             self.template_args["args"] = f'["-s", "-p", "{self.port}"]'
 
-        perf.PerfServer.__init__(self, tc, ts)
```

Comment:
This was intentional because we need to have template_args set before calling the parent class's init.

Reply:
ah. Well, it doesn't work, because at this point, `self.template_args` does not exist, and you'd get an exception. This change avoids the crash.

Indeed, `perf.PerfServer.__init__()` will already render the file, and it needs those template arguments. 

On [`th/next`](https://github.com/thom311/ocp-traffic-flow-tests/commits/th/next/) branch I already reworke that further (see commit "task: extract initialize() method").

I'd prefer if we could merge this commit as-is (at least it doesn't crash right away -- it only fails to template the values).

This will be fixed later (after this MR, with patches from `th/next` branch.

@wizhaoredhat OK?

Reply:
This is ok with me. Thanks!